source/views/calendar/event-detail.js

```diff
@@ -59,7 +90,8 @@ export function EventDetail(props: {
         <MaybeSection header="EVENT" content={title} />
         <MaybeSection header="TIME" content={times} />
         <MaybeSection header="LOCATION" content={location} />
-        <MaybeSection header="DESCRIPTION" content={summary} />
+        <MaybeSection header="DESCRIPTION" content={trimmedSummary} />
+        <Links header="LINKS" content={summary} />
```

Comment:
Rename these variables maybe? Store the plain summary in `summary` and the one to be linked in… idk. I don’t think I have a good name for it. `rawSummary`?

My objection here is that you didn’t really change the content of the DESCRIPTION one, but the line changed? Idk. I guess you can ignore this comment if you want.